sale_partner_shipping_default_partner_invoice/views/res_partner_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">res.partner.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_sending_method']" position="before">
+                <field name="default_invoice_partner_id" />
+            </xpath>
+            <xpath
+                expr="//field[@name='child_ids']//form//field[@name='comment']"
+                position="after"
+            >
+                <field name="default_invoice_partner_id" />
+            </xpath>
```

Comment:
Can we show the field right below the address section, as the significance of maintaining this information is rather big when you install this module, and we don't want to have the field placed somewhere not so obvious?